platform-hub-auth-proxy/README.md

```diff
@@ -4,7 +4,11 @@ Uses [keycloak-proxy](https://github.com/gambol99/keycloak-proxy)
 
 The `local` folder contains scripts to manage a local `keycloak-proxy` instance in a Docker container.
 
+**IMPORTANT:** to allow the Docker container to talk to the API running on your host: set up an alias on your local loopback adapter using `sudo ifconfig lo0 alias 10.200.10.1/24`.
```

Comment:
Yeah, I'm not 100% happy with this solution TBH, but it's intended to be a quick and mostly viable solution for local dev.

I'm not keen on a fully docker compose or minikube setup as I've found that things like auto reload of code, etc. don't work so well when developing on your machine but running the services in a containerised or VM environment. Right now I'm only running things in containers that are essentially just external dependencies (Postgres and the keycloak-proxy).

Will quiz you on setting a local Docker network, at some point :)